<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GCButton</class>
 <widget class="QFrame" name="GCButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>105</width>
    <height>105</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>105</width>
    <height>105</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>105</width>
    <height>105</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>GCButton</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::WinPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QLabel" name="hotKeyLbl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="iconLbl">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>96</width>
     <height>64</height>
    </rect>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="titleLbl">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>70</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gcom.qrc"/>
 </resources>
 <connections/>
</ui>
